modules/salt/pages/scale-minions.adoc

```diff
@@ -33,6 +33,11 @@ For larger installations, this number may be increased by editing the following
 LimitNOFILE=16384
 ----
 
+////
+This content is superceded by the content in modules/salt/pages/salt-rate-limiting.adoc
```

Comment:
This is not really superseded, in fact https://github.com/SUSE/doc-susemanager/blob/master/modules/salt/pages/salt-rate-limiting.adoc points here:

```adoc
For more information on minion timeouts, see xref:scale-minions.adoc[].
```

I think it should not be commented out.

Reply:
I haven't deprecated the whole section, just the bit about timeouts, which exists in the rate-limiting page.

Reply:
Sorry, unless I am mistaken we are commenting too much here:

 - [salt-rate-limiting.adoc](https://github.com/SUSE/doc-susemanager/blob/master/modules/salt/pages/salt-rate-limiting.adoc#presence-ping-timeout) explains `presence_ping_timeout` / `presence_ping_gather_job_timeout`, basically stating that they work like `timeout`/`gather_job_timeout`. But it does not go into much detail over how the latter two work, pointing here instead
 - this page explains:
    - `timeout`/`gather_job_timeout` in general ("Salt Timeouts", "Background Information" titles). This part should be **kept**
    - the now-removed presence ping implementation: parameters were named `salt_presence_ping_timeout`/`salt_presence_ping_gather_job_timeout` and they worked similarly to `presence_ping_timeout` / `presence_ping_gather_job_timeout` ("A Presence Ping Mechanism for Unreachable Salt Minions" and "Overriding Salt Presence Timeout Values" titles). This part can be **removed**

Does this make sense?

Reply:
Ah ha! Thanks for your patience. Updated now :)